### PR TITLE
audit results for C4G-18

### PR DESCRIPTION
**Pull request by edwd42**
- [X] Jira ticket number [C4G-18](https://jira.sdlc.redcross.us/browse/C4G-18)
- [X] Created file results_`C4G-18`_`IN`in `/public/scan-results/data`

### Issues with some links 
- [ ] Skip some URL due an Error (List Included)
- [X] Set Label `Missing_URL` for future reference

### CODE REVIEW

File: public/tools/auto-audit/auto-audit.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 start_row = max(args.start_at, 2)
 if start_row < sheet0.max_row:
-    for i in range(start_row, sheet0.max_row):
+    for i in range(start_row, sheet0.max_row + 1):
```
Comment (bhaumikmistry) [inline public/tools/auto-audit/auto-audit.py:35]: Was there any issue? any comment on what is the fix?

Comment (edwd42) [reply public/tools/auto-audit/auto-audit.py:35]: yes, the script was quitting at row 50, but I had one more row to go in the spreadsheet, then I realized the range() function stop parameter is index number which is one less than the stop number

**Comment by bhaumikmistry** [comment]
Before You merge, I would suggest to Rebase your branch on master so that the new Github CI Action runs to check the file names.

**Comment by edwd42** [comment]
Rebase always freaks me out.  It's fine on my own repo, rebase all day long...  But on a collaboration repo, hmmmm.  I'm not sure how to rebase my branch on master like you suggest.  And this SO post makes me apprehensive about rebaseing.  Could you guide me through this so I don't blow things up?  Thanks!

**Pull request merged**